docs/reference/inference/service-elser.asciidoc

```diff
@@ -63,6 +66,9 @@ The following example shows how to create an {infer} endpoint called
 `my-elser-model` to perform a `sparse_embedding` task type.
 Refer to the {ml-docs}/ml-nlp-elser.html[ELSER model documentation] for more info.
 
+The request below will automatically download and deploy the ELSER model if it
+isn't already downloaded and deployed.
```

Comment:
```suggestion
The request below will automatically download the ELSER model if it isn't already downloaded and deploy then model
```

If the model is already deployed this request creates a new deployment whatever it doesn't reuse an existing one which is one way of interpreting `isn't already downloaded and deployed.`

Reply:
@davidkyle Updated via [ee59f0f](https://github.com/elastic/elasticsearch/pull/111278/commits/ee59f0fcae5feb6cb0b03637c9e5ce73692d21ee).